### DISCUSSION TITLE
For Albania iptv

### DISCUSSION THREAD

**Discussion by renimela** [iptv-org/community#424] [Q&A] [answered]
Hello. In the playlist "Albania" it has been some time that some channels are missing. Will they be fixed ?? !!
For example: Tring Fantasy, Top Channel, Klan etc etc ... Thnx

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream
